.travis.yml

```diff
@@ -7,9 +7,12 @@ language: objective-c
 env:
   matrix:
     
-    - CONDA_PY=27  
-    - CONDA_PY=34  
-    - CONDA_PY=35  
+    - CONDA_NPY=110  CONDA_PY=27
```

Comment:
@jakirkham - I just tried and it seems that conda-smithy doesn't need the tokens (though it does tell you you don't have them). That is late ones for you :sleeping: :smile: